packages/lesmis/src/views/ExportView.js

```diff
@@ -90,15 +93,18 @@ const getChildren = ({
 export const ExportView = ({ viewProps, viewType, className }) => {
   const { getExtraExportViewProps, PageContainer, PageContent } = EXPORT_VIEWS[viewType];
   const exportViewProps = { ...viewProps, ...getExtraExportViewProps() };
-
+  const { selectedDashboard } = exportViewProps;
   const { dropdownOptions } = useDashboardDropdownOptions();
```

Comment:
```suggestion
  const { selectedOption } = useDashboardDropdownOptions();
```

We can get the `selectedOption` here, no need to find the selected dashboard below.